app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    first_file = command[1]
+    second_file = command[2]
+    if len(os.path.dirname(second_file)) > 0:
+        os.makedirs(os.path.dirname(command[2]), exist_ok=True)
+        with open(second_file, "w") as f:
+            f.write("")
+        os.remove(os.path.dirname(second_file) + "\\" + first_file)
```

Comment:
firstly, created path will not work for some OS
secondly, what do you do here at all?

Reply:
rewritten again